src/main/java/com/gluonhq/richtext/model/Piece.java

```diff
@@ -88,4 +91,19 @@ public String toString() {
                 ", decoration=" + decoration +
                 '}';
     }
+
+    private TextDecoration normalizeDecoration(TextDecoration decoration) {
```

Comment:
I'd move this method to `TextDecoration` to keep `Piece` more clean.

Reply:
Agree